tests/integration/api_v3/http/posRewardsLocked.test.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../config');
+const { api } = require('../../../helpers/api');
+
+const baseUrl = config.SERVICE_ENDPOINT;
+const baseUrlV3 = `${baseUrl}/api/v3`;
+
+const {
+	badRequestSchema,
+} = require('../../../schemas/httpGenerics.schema');
+
+const {
+	goodResponseSchema,
+} = require('../../../schemas/api_v3/posRewardsLocked.schema');
+
+const endpoint = `${baseUrlV3}/pos/rewards/locked`;
+const stakesEndpoint = `${baseUrlV3}/pos/stakes`;
+
+// TODO: Enable test when pos/stakes endpoint is available
+xdescribe('Rewards Locked API', () => {
```

Comment:
Let the tests fail for now. Please keep the tests enabled.

Reply:
Applied.